client/fe/webpack/plugins.js

```diff
@@ -100,7 +100,7 @@ const webpackDefaultPlugins = [
 
   // Provide configured Vue and Bus instances to all modules
```

Comment:
This is not correct anymore... 

```suggestion
  // Provide Vue instance to all modules (is configured in InitVue.js before initialization).
```

Reply:
changed in 6852b8a27abb59355b169831fc0863f1736e9ab6